index.js

```diff
@@ -53,9 +62,10 @@ app.get('/favicon.ico', (req, res) => {
 });
 
 app.all('/graphql', (req, res) => res.redirect('/'));
+auth(app);
```

Comment:
I don't think we want to change this. E.g. Anonymous users should be able to browse Force.